mod/01_core/set/all/content.rb

```diff
@@ -19,8 +19,11 @@ def raw_content
   structure ? template.db_content : db_content
 end
 
-def chunk_list #override to customize by set
-  :default
+
+format :all do
+  def chunk_list #override to customize by set
+    :default
+  end
```

Comment:
It's pretty clear to me that format(:all) is a valuable API option.  As to whether it's needed here,  I do agree that it would be nice to have a test case that fails without it.

It's pretty important to get this merged in to stop the email subject problem (though for now I changed *title on Wikirate.org to "Wikirate" from "Wikirate.org", which should buy us a little time)